roles/cinder-data/tasks/main.yml

```diff
@@ -39,6 +39,10 @@
 - name: iscsi start
   service: name=open-iscsi state=started
 
+- include: vnx.yml
```

Comment:
Many backends support multipath, that doesn't seem to be VNX specific, and it seems like we could include that in the core role, or perhaps a common include for backends which support it.